vispy/visuals/volume.py

```diff
@@ -707,6 +707,56 @@ def interpolation(self, interp):
             self._texture.interpolation = self._interpolation
             self.update()
 
+    @staticmethod
+    @lru_cache(maxsize=None)
```

Comment:
There should probably be some upper limit on this.

Reply:
What would you consider reasonable? Keep in mind that this caches a new result depending on the *number* of clipping planes, not what the clipping planes look like. I don't expect anyone to reasonably go above 10 clipping planes, so we can cache about 10?

Reply:
I guess 10 works for me. I could see less, but I am not the targeted user for this so I'm not sure. Just wanted to avoid a "memory leak" by leaving this unbounded. 10 works.